src/server/services/crmService/index.js

```diff
@@ -1,15 +1,6 @@
-import fetch from 'isomorphic-fetch'
-
-import config from 'src/config'
-
-const properties = {
-  echoSignUp: 'signed_up_for_echo',
-}
-
-const paths = {
-  contactProfileByEmail: email => `/contacts/v1/contact/email/${email}/profile`,
-  contactProfileByVID: vid => `/contacts/v1/contact/vid/${vid}/profile`,
-}
+import {default as getContactByEmail} from './getContactByEmail'
```

Comment:
Do we have a linter that prefers:

```js
import {default as getContactByEmail} from './getContactByEmail'
```

over:

```js
import getContactByEmail from './getContactByEmail'
```

Reply:
Yes we do. I think it's xo